### PR TITLE
SVITSMS send sms

### PR DESCRIPTION
**Pull request by lubo4kaTarasenko**
e733f5b7-3648-415a-a8bd-85678224a0ed

### CODE REVIEW

File: app/models/user_phone.rb [hunk 1]
```diff
@@ -12,6 +12,10 @@ def confirmed?
   private
 
   def generate_code
-    self.code = rand(1000..9999)
+    if Rails.env.production?
+      self.code = rand(1000..9999)
+    else
+      self.code = 1111
+    end
```
Comment (galulex) [inline app/models/user_phone.rb]: `self.code = Rails.env.production? ? rand(1000..9999) : 1111`

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -8,6 +9,10 @@ def create
     @user ||= User.create(user_params)
     if @user.valid?
       session[:phone] = @user.phones.first.number
+      if Rails.env.production?
```
Comment (galulex) [inline app/controllers/sessions_controller.rb]: Move this logic to SmsSender class

Comment (lubo4kaTarasenko) [reply app/controllers/sessions_controller.rb]: done

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -8,6 +9,8 @@ def create
     @user ||= User.create(user_params)
     if @user.valid?
       session[:phone] = @user.phones.first.number
+      sms = SmsSender.new(@user.phones.first.number, @user.phones.first.code,"380680435966", "nananabananananana")
```
Comment (galulex) [inline app/controllers/sessions_controller.rb]: Move login and password. To rails credentials. [Click](https://www.engineyard.com/blog/rails-encrypted-credentials-on-rails-5.2)

File: lib/sms_sender.rb [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    }
+    xml = <<~XML
+    <?xml version="1.0" encoding="utf-8"?>
+    <request>
+      <operation>SENDSMS</operation>
+      <message start_time="AUTO" end_time="AUTO" lifetime="4" rate="120" desc="My first campaign " source="SMS">
+        <body>#{text}</body> 
+        <recipient>#{phone}</recipient>
+      </message>
+    </request>
```
Comment (galulex) [inline lib/sms_sender.rb]: Move it to constant

Comment (lubo4kaTarasenko) [reply lib/sms_sender.rb]: what to constant? uri is the only one thing, i can move to a constant, everything else is dynamic

Comment (galulex) [reply lib/sms_sender.rb]: XML. Check example [here](https://gist.github.com/galulex/ee6d057f27eff6375b9f#file-lesson1-rb-L3)

Comment (lubo4kaTarasenko) [reply lib/sms_sender.rb]: everything is done

**Pull request merged**